src/libstd/sys/unix/c.rs

```diff
@@ -15,11 +15,26 @@
 
 pub use self::select::fd_set;
 pub use self::signal::{sigaction, siginfo, sigset_t};
+#[cfg(not(target_os = "nacl"))]
 pub use self::signal::{SA_ONSTACK, SA_RESTART, SA_RESETHAND, SA_NOCLDSTOP};
```

Comment:
Well we don't have a concrete convention here and it avoids `#[cfg]` spaghetti glue, so let's use globs in this use case.